OpenSim/Utilities/simmToOpenSim/detools.c

```diff
@@ -809,7 +809,6 @@ XForm* get_deform_xform (DeformObject* dfm, int deformMode)
       case DE_DEFORM_END_MODE:
          return &dfm->deform_end;
    }
-/*   assert(0);*/
```

Comment:
Changes in this file are drive-bys on comments containing the regex `\bassert\(` (i.e. find all uses of bare `assert`)